<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="5" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="80453" started-at="2017-07-13T04:35:49Z" finished-at="2017-07-13T04:37:09Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="80453" started-at="2017-07-13T04:35:49Z" finished-at="2017-07-13T04:37:09Z">
      <class name="testcases.VerifyEmail">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="beforeTest" is-config="true" duration-ms="721" started-at="2017-07-13T10:05:49Z" finished-at="2017-07-13T10:05:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="testBrowser()[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="testBrowser" is-config="true" duration-ms="14175" started-at="2017-07-13T10:05:49Z" finished-at="2017-07-13T10:06:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBrowser -->
        <test-method status="PASS" signature="testVerifySendingEmail()[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="testVerifySendingEmail" duration-ms="14711" started-at="2017-07-13T10:06:04Z" finished-at="2017-07-13T10:06:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testVerifySendingEmail -->
        <test-method status="PASS" signature="TestProcessingOfEmail()[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="TestProcessingOfEmail" duration-ms="9074" started-at="2017-07-13T10:06:18Z" finished-at="2017-07-13T10:06:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestProcessingOfEmail -->
        <test-method status="PASS" signature="TestUnProcessingOfEmail()[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="TestUnProcessingOfEmail" duration-ms="5525" started-at="2017-07-13T10:06:27Z" finished-at="2017-07-13T10:06:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestUnProcessingOfEmail -->
        <test-method status="PASS" signature="TestBatchProcessingOfEmail()[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="TestBatchProcessingOfEmail" duration-ms="19011" started-at="2017-07-13T10:06:33Z" finished-at="2017-07-13T10:06:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestBatchProcessingOfEmail -->
        <test-method status="PASS" signature="TestBatchUnProcessingOfEmail()[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="TestBatchUnProcessingOfEmail" duration-ms="16099" started-at="2017-07-13T10:06:52Z" finished-at="2017-07-13T10:07:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestBatchUnProcessingOfEmail -->
        <test-method status="PASS" signature="postCondition()[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="postCondition" is-config="true" duration-ms="1086" started-at="2017-07-13T10:07:08Z" finished-at="2017-07-13T10:07:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
        <test-method status="FAIL" signature="getResult(org.testng.ITestResult)[pri:0, instance:testcases.VerifyEmail@1bce4f0a]" name="getResult" is-config="true" duration-ms="0" started-at="2017-07-13T10:07:09Z" finished-at="2017-07-13T10:07:09Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[wrong number of arguments]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.afterRun(TestRunner.java:1021)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- testcases.VerifyEmail -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
